pwa/app/components/ui/carousel.tsx

```diff
@@ -124,8 +124,7 @@ const Carousel = React.forwardRef<
           carouselRef,
           api: api,
           opts,
-          orientation:
-            orientation || (opts?.axis === 'y' ? 'vertical' : 'horizontal'),
+          orientation,
```

Comment:
This seems correct from a typing perspective (`orientation` seems to never be null), but it still spooks me that the original component had this extra logic @tervay